marklogic-data-hub-central/ui/src/components/modeling/graph-view/graph-view.tsx

```diff
@@ -22,16 +23,60 @@ type Props = {
 const GraphView: React.FC<Props> = (props) => {
 
   const {modelingOptions, setSelectedEntity, closeSidePanelInGraphView} = useContext(ModelingContext);
+  const [viewSidePanel, setViewSidePanel] = useState(false);
+  const [nodeSuggested, setNodeSuggested] = useState(["a"]);
+  const [entitySelected, setEntitySelected] = useState("");
+  const [isEntitySelected, setIsEntitySelected] = useState(false);
+
+  useEffect(() => {
```

Comment:
This useEffect was removed in the previous PR for deletion if entity. Any particular reason why we are using this again along with viewSidePanel state variable, which is moved to the context file now.

Reply:
It is not being used anywhere; I have removed them